### PR TITLE
feat: add new filters for dashboards

### PR DESCRIPTION
**Pull request by MKhoaaa**
- Add a transition for 'setPane' method
- Add a filter for sorting item's rental type
- Add 'ALL' option to the user's role filter and make it compatible with the searching function
- Align table columns to center left
- Minor UI changes for the 'Help' button

### CODE REVIEW

File: src/main/java/pls_no_shinobu/videostore/controller/AdminDashboardController.java [hunk 1]
```diff
@@ -38,16 +38,51 @@ private enum SearchBy {
...
+        public Item.RentalType getType() {
+            return type;
+        }
+    }
+
+    private enum Role {
+        ALL(null),
+        REGULAR(User.UserType.REGULAR),
+        VIP(User.UserType.VIP),
+        ADMIN(User.UserType.ADMIN);
```
[the remark this answers is not in the document]
Comment (MKhoaaa) [reply src/main/java/pls_no_shinobu/videostore/controller/AdminDashboardController.java:62]: My mistake, it it missing. I will add `User.UserType.GUEST`

**Comment by catouberos** [comment]
The "ALL" filter seems to have some bugs, can you recheck it again?

https://github.com/catouberos/video-store/assets/12823486/a84b2faf-18f2-4a3a-bbbb-5605b55470d3

**Comment by MKhoaaa** [comment]
> The "ALL" filter seems to have some bugs, can you recheck it again?
> 
>  Screen.Recording.2023-05-26.at.23.08.37.mov

I percieved 2 issues as in the recording:

1) Selecting options in the filters doesn't check the value in the search bar -> I have fixed this
2) Searching 'admin' doesn't returns any users -> I believe this is because while we are searching by 'Name', this admin account doesn't have a name. Can you check and give comment on this case @catouberos?

**Comment by catouberos** [comment]
>     1. Selecting options in the filters doesn't check the value in the search bar -> I have fixed this

This seems to work now, thank you.

>     2. Searching 'admin' doesn't returns any users -> I believe this is because while we are searching by 'Name', this admin account doesn't have a name. Can you check and give comment on this case @catouberos?

I didn't recognise this at first, my bad.

**Pull request merged**